.github/workflows/main.yml

```diff
@@ -50,19 +65,11 @@ jobs:
     - name: Set up python 3.x
       uses: actions/setup-python@v2
       with:
-        python-version: 3.6
+        python-version: ${{ matrix.python }}
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
-    - name: Run tests
-      run: |
-        python -m unittest discover tests -v
-    - name: Generate coverage report
-      run: |
-        pip install pytest
-        pip install pytest-cov
-        pytest --cov=./ --cov-report=xml
- 
-    #- name: Upload coverage to Codecov
-          
+    - name: Run tests and generate coverage report
+      run: coverage run -m unittest discover tests -v 
+    #- name: Upload coverage to Codecov
```

Comment:
This comment should be removed

Reply:
It's kind of like a todo actually.
So maybe you could just add TODO before the comment that would be nice @vj-codes.